controllers/homeRoutes.js

```diff
@@ -2,15 +2,19 @@ const router = require('express').Router();
 const { Goal, Todo } = require('../models');
 // const withAuth = require('../utils/auth');
 
+router.get('/', async (req, res) => {
+  res.render('homepage', { logged_in: req.session.logged_in });
```

Comment:
@Mulloa90 Do we need to set 'login' if not logged in or does it render as the default?